plugins/module_utils/dd_statement.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) IBM Corporation 2020
+# Copyright (c) IBM Corporation 2020, 2023
```

Comment:
maybe 2020-2024?

Reply:
Hey Rich, thanks for looking out for copyrights, most if not all this content was from 2023, template.py's last change was 4 months ago.

BTW, unrelated to your comment, copyrights should be in this form: YEAR1, YEAR2 where YEAR2 is the last year it was updated.

Pulled from our channel:
```
Regarding the copyright year commas 2020, 2021, 2023 vs 2020 - 2023 the suggestion is that it should contain 2 years, the start year where its original creation, and an end year being the last year it was modified, so it should be 2020,2023 
```